### PR TITLE
chore: header parameter in the test

### PR DESCRIPTION
**Pull request by krol3**
Adding header as a parameter in the tests.

### CODE REVIEW

File: outputs/http_test.go [hunk 1]
```diff
@@ -28,22 +28,35 @@ func TestHTTPClient_Send(t *testing.T) {
...
+			testServerFunc: func(w http.ResponseWriter, r *http.Request) {
+				assert.Equal(t, "bar value", r.Header.Get("fookey"))
+				assert.Equal(t, "", r.Header.Get("POSTEE_EVENT"))
+				fmt.Fprintln(w, "Hello, client")
+			},
+		},
+		{
+			name:    "happy path method get",
+			method:  http.MethodGet,
+			headers: map[string]string{"description": "foo bar baz header"},
```
Comment (AndreyLevchenko) [inline outputs/http_test.go:48]: naming is a bit confusing. It's actually not `headers` but result of template evaluation.  It's called `contents`/`data`/`input` in existing implementation of `Send(...)` method

File: outputs/http_test.go [hunk 2]
```diff
@@ -28,22 +28,35 @@ func TestHTTPClient_Send(t *testing.T) {
...
+		headers        map[string]string
 		testServerFunc http.HandlerFunc
 		expectedError  string
 	}{
 		{
-			name:   "happy path method get",
-			method: http.MethodGet,
+			name:    "none headers in send method",
+			method:  http.MethodGet,
+			headers: nil,
```
Comment (AndreyLevchenko) [inline outputs/http_test.go:38]: From what I can understand it's not a real use case. There is no option to configure template to return `nil` as result of evaluation.

**Comment by simar7** [comment]
hi @krol3 - I've already addressed the assertion in this PR https://github.com/aquasecurity/postee/pull/300 and a bug fix.

**Comment by simar7** [comment]
closing in favour of https://github.com/aquasecurity/postee/pull/300

**Pull request closed without merging**